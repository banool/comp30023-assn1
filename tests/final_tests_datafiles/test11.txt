0 1 10 60
0 2 4096 60
0 3 10 60
60 4 12288 10
180 5 4096 10
240 6 16384 60
300 7 6144 10
301 8 10 5
302 9 10 5
