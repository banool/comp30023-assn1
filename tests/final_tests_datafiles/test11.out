time 0, 1 running, numprocesses=1, numholes=1, memusage=1%
time 2, 2 running, numprocesses=2, numholes=1, memusage=26%
time 4, 3 running, numprocesses=3, numholes=1, memusage=26%
time 6, 1 running, numprocesses=3, numholes=1, memusage=26%
time 10, 2 running, numprocesses=3, numholes=1, memusage=26%
time 14, 3 running, numprocesses=3, numholes=1, memusage=26%
time 18, 1 running, numprocesses=3, numholes=1, memusage=26%
time 26, 2 running, numprocesses=3, numholes=1, memusage=26%
time 34, 3 running, numprocesses=3, numholes=1, memusage=26%
time 42, 1 running, numprocesses=3, numholes=1, memusage=26%
time 50, 2 running, numprocesses=3, numholes=1, memusage=26%
time 58, 3 running, numprocesses=3, numholes=1, memusage=26%
time 66, 4 running, numprocesses=1, numholes=1, memusage=75%
time 68, 4 running, numprocesses=1, numholes=1, memusage=75%
time 72, 1 running, numprocesses=2, numholes=1, memusage=76%
time 80, 2 running, numprocesses=2, numholes=2, memusage=26%
time 88, 3 running, numprocesses=3, numholes=2, memusage=26%
time 96, 4 running, numprocesses=1, numholes=1, memusage=75%
time 100, 1 running, numprocesses=1, numholes=1, memusage=1%
time 108, 2 running, numprocesses=2, numholes=1, memusage=26%
time 116, 3 running, numprocesses=3, numholes=1, memusage=26%
time 124, 1 running, numprocesses=3, numholes=1, memusage=26%
time 132, 2 running, numprocesses=3, numholes=1, memusage=26%
time 140, 3 running, numprocesses=3, numholes=1, memusage=26%
time 148, 1 running, numprocesses=3, numholes=1, memusage=26%
time 156, 2 running, numprocesses=3, numholes=1, memusage=26%
time 164, 3 running, numprocesses=3, numholes=1, memusage=26%
time 172, 1 running, numprocesses=3, numholes=1, memusage=26%
time 178, 2 running, numprocesses=2, numholes=2, memusage=26%
time 184, 5 running, numprocesses=2, numholes=2, memusage=26%
time 186, 5 running, numprocesses=2, numholes=2, memusage=26%
time 190, 3 running, numprocesses=2, numholes=2, memusage=26%
time 196, 5 running, numprocesses=1, numholes=1, memusage=25%
time 240, 6 running, numprocesses=1, numholes=0, memusage=100%
time 242, 6 running, numprocesses=1, numholes=0, memusage=100%
time 246, 6 running, numprocesses=1, numholes=0, memusage=100%
time 254, 6 running, numprocesses=1, numholes=0, memusage=100%
time 262, 6 running, numprocesses=1, numholes=0, memusage=100%
time 270, 6 running, numprocesses=1, numholes=0, memusage=100%
time 278, 6 running, numprocesses=1, numholes=0, memusage=100%
time 286, 6 running, numprocesses=1, numholes=0, memusage=100%
time 294, 6 running, numprocesses=1, numholes=0, memusage=100%
time 300, 7 running, numprocesses=1, numholes=1, memusage=38%
time 302, 8 running, numprocesses=2, numholes=1, memusage=38%
time 304, 9 running, numprocesses=3, numholes=1, memusage=38%
time 306, 7 running, numprocesses=3, numholes=1, memusage=38%
time 310, 8 running, numprocesses=3, numholes=1, memusage=38%
time 313, 9 running, numprocesses=2, numholes=2, memusage=38%
time 316, 7 running, numprocesses=1, numholes=1, memusage=38%
time 320, simulation finished.
